static/scss/not-found.scss

```diff
@@ -0,0 +1,15 @@
+.not-found {
```

Comment:
Is there a design for this?

Reply:
![image_1024_720](https://user-images.githubusercontent.com/6207644/34676895-5ec5c432-f45c-11e7-953d-9af0539b7604.png)

Reply:
(I don't think this mock is posted on invision yet)